setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='wagtailcommonblocks',
-    version='0.0.2',
+    version='0.0.3',
     packages=['commonblocks'],
     include_package_data=True,
     license='BSD License',
```

Comment:
## Bonus points

Looking at this metadata, it would be nice if:

- The repository was renamed to match the name of the package (`wagtailcommonblocks`).
- License = MIT
- URL is wrong - needs to either point to the current repo, or the repo needs renaming.
- etc

If you have the time to look at those, I would suggest looking at what https://github.com/springload/wagtaildraftail does. It's our most up to date example of best-practice Python module publishing.